docker-files-for-Gitlab-CI-rust/parity-eth-linux-docs/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:xenial as builder
+
+# metadata
+LABEL maintainer="devops-team@parity.io" \
+	vendor="Parity Technologies" \
+	name="parity/rust-parity-ethereum-docs" \
+	description="Image for CI docs stage for Parity Ethereum." \
+	url="https://gitlab.com/paritytech/scripts/" \
+	vcs-url="scripts/docker-files-for-Gitlab-CI-rust/parity-eth-linux-docs"
+
+WORKDIR /builds/parity/parity-ethereum/
+
+# install tools and dependencies
+RUN set -eux; \
+	apt-get -y update; \
+    apt-get install -y --no-install-recommends \
+		curl ca-certificates git; \
+# install nodejs and yarn
+	curl -sL https://deb.nodesource.com/setup_10.x | bash -; \
+	curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -; \
+	echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list; \
+	apt-get -y update; \
```

Comment:
is it intentional to run it twice?

Reply:
unfortunately yes, first time to be able to install anything, and this update is to fetch info from those new repositories

Reply:
why not add repos at the top?

Reply:
there were no curl and certs before!

Reply:
you could put just the echo line on top - but maybe this small speedup will hurt legibility more

Reply:
this image won't likely to be rebuilt often

Reply:
but I understand, will think how to optimize